ports/nrf/Makefile

```diff
@@ -38,9 +38,6 @@ endif
 # qstr definitions (must come before including py.mk)
 QSTR_DEFS = qstrdefsport.h $(BUILD)/pins_qstr.h
 
-# MicroPython feature configurations
-MICROPY_ROM_TEXT_COMPRESSION ?= 1
-
```

Comment:
Ah, sorry, when I suggested to move the change to down below I didn't realise you'd move this part as well.  The idea is to keep "MicroPython feature configurations" in this location in the Makefile, just before including `py/py.mk` because that latter file may need to inspect these feature configuration variables (not for `MICROPY_ROM_TEXT_COMPRESSION` but it's still good to define them early on in the Makefile).  Other ports do it this way.

So, I think it should stay here and be something like:
```
ifeq ($(DEBUG), 0)
MICROPY_ROM_TEXT_COMPRESSION ?= 1
endif
```